baselines/tamuna/pyproject.toml

```diff
@@ -0,0 +1,144 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "tamuna"
+version = "1.0.0"
+description = "Flower Baselines"
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.dev>"]
+readme = "README.md"
+homepage = "https://flower.dev"
+repository = "https://github.com/adap/flower"
+documentation = "https://flower.dev"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Typing :: Typed",
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8.15, <3.12.0"  # don't change this
+flwr = "1.3.0" # don't change this
+ray = "1.11.1" # don't change this
+hydra-core = "1.3.2" # don't change this
+matplotlib = "3.7.1"
+torch = {version = "2.0.1+cu118", source = "pytorch"}
+torchvision = {version = "0.15.2+cu118", source = "pytorch"}
+torchaudio = {version = "2.0.2+cu118", source = "pytorch"}
```

Comment:
At least on my machine this was causing the download of these packages for all platforms (not just linux, but also windows... taking really a long time).  Did you also experience this? Would it be ok to replace these lines with the following?
```
torch = { url = "https://download.pytorch.org/whl/cu118/torch-2.0.1%2Bcu118-cp39-cp39-linux_x86_64.whl"}
torchvision = { url = "https://download.pytorch.org/whl/cu118/torchvision-0.15.2%2Bcu118-cp39-cp39-linux_x86_64.whl"}
torchaudio = { url = "https://download.pytorch.org/whl/cu118/torchaudio-2.0.2%2Bcu118-cp39-cp39-linux_x86_64.whl"}
```
And then remove the `pytorch` source a few lines below?

Reply:
This would work nice for Linux, but Windows packages would not be available and this would not work on Windows, right? If you plan to target mainly Linux, then this would be okay.

Reply:
done